src/components/InvoicesList/InvoicesList.test.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { render, cleanup, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import IntlProvider from '../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import { AppServicesProvider } from '../../services/pure-di';
+import { BrowserRouter } from 'react-router-dom';
+import InvoicesList from './InvoicesList';
+
+describe('InvoiceList component', () => {
+  afterEach(cleanup);
+
+  it('should show invocies', async () => {
+    // Arrange
+    const invoicesCollection = {
+      items: [
+        {
+          accountId: 'CD0000000073689',
+          product: 'Prod 1',
+          date: '09/25/2020',
```

Comment:
It is not a realistic expected result

Reply:
i forgot the replace the date in the test, thanks @andresmoschini . I will replace the format in the test

Reply:
Done